crates/vm/levm/src/gas_cost.rs

```diff
@@ -449,18 +453,40 @@ pub fn sstore(
     } else {
         let static_gas = SSTORE_STATIC;
 
+        let default_dynamic = match fork {
+            // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1283.md
+            Fork::Constantinople => SSTORE_DEFAULT_CONSTANTINOPLE,
+            // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2200.md
+            _f if fork == Fork::Istanbul || fork == Fork::MuirGlacier => {
```

Comment:
Quick question (and more of a nitpick really)
Why keep the `_f` Why not 
```suggestion
            f if f == Fork::Istanbul || f == Fork::MuirGlacier => {
```